code/datums/helper_datums/command_alerts.dm

```diff
@@ -581,4 +581,17 @@ The access requirements on the Asteroid Shuttles' consoles have now been revoked
 
 /datum/command_alert/nuclear_operatives/announce()
 	message = "Presence of hostile Syndicate operatives has been confirmed in the vicinity of [station_name()]. Command staff is advised to monitor the status of all high-value assets, and security staff should co-operate with all crew members in securing the station from infiltration."
-	..()
\ No newline at end of file
+	..()
+
+//////////////////////BLACK MARKET
+
+/datum/command_alert/ian_sold
+	name = "Ian has been sold - Urgent Alert"
+	alert_title = "Ian Sold"
+	force_report = 1
+
+/datum/command_alert/New(var/mob/killer = null)
+	if(killer)
+		message = "A regular bluespace scan of important lifeforms across the Centcomm sphere of influence has revealed that Ian has been sold to black market dealers. This is greatly worrying, as we are unable to ensure his safety nor retrieve him. Investigation has revealed that [killer.name] is responsible. You monster."
```

Comment:
this is five kinds of over the top

Reply:
how so? It's entertaining, fits roleplay, and is far more in character than most of the adminbusses *cough* that take place on a daily basis.

Reply:
Maybe something more of a generic message on the lines of "we have detected an unauthorised bluespace transport of live matter to/from the station, this is a breach of (some bullshit space law)"
That way it can cover all pets, and also eventually buying animals

Reply:
>entertaining, fits roleplay and more in character than bus
"hey this one fucking pet on your station is EXTREMELY important, so important in fact that we will now show you the valids; [valids]. please go acquire the valids."

why the fuck would centcom care about a corgi

Reply:
I changed the message to imply that Centcomm was investigating a signal and noticed Ian was teleported, who happens to be a person of interest as submitted by our station, meaning that they're obligated to send a report.

Reply:
Though your new explanation is cuter and fits more, it's still over the top to send a command report over a stolen corgi, *especially if you include the would-be kidnapper's identity*.

You're inciting the entire crew to cascade upon that individual to valid them. Over a simple mob that only the HoP cares about.